File: CPP/Math/Convert Integer to the Sum of Two No-Zero Integers.cpp
class Solution {
public:
    bool isContainZero(int n) {
        while (n > 0) {
            if (n % 10 == 0) {
                return false;
            }
            n = n / 10;
        }
        return true;
    }
    vector<int> getNoZeroIntegers(int n) {
        for (int a = 1; a < n; a++) {
            int b = n - a;
            if (isContainZero(a) && isContainZero(b)) {
                return {a, b};
            }
        }
        return {};
    }
};

File: Python/Math/Convert Integer to the Sum of Two No-Zero Integers.py
class Solution(object):
    def isContainZero(self,n):
        while n>0:
            if n%10==0:
                return False
            n//=10
        return True
    
    def getNoZeroIntegers(self, n):
        """
        :type n: int
        :rtype: List[int]
        """
        for a in range(1,n):
            b=n-a
            if self.isContainZero(a) and self.isContainZero(b):
                return [a,b]
        return []
